fuzz/fuzz_targets/scrypt_fuzzer.rs

```diff
@@ -0,0 +1,150 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#![no_main]
+
+use libfuzzer_sys::fuzz_target;
+use scrypt::{scrypt, Params};
+
+#[cfg(feature = "simple")]
```

Comment:
These `cfg`s are no longer needed. Even worse, I think the crate will not even compile with these. Can you add a CI job for the `fuzz` crate, similar to the [`benches` job](https://github.com/RustCrypto/stream-ciphers/blob/master/.github/workflows/benches.yml)?

Reply:
I have fixed the logic and import. I have also add a CI for testing the fuzz build with cargo fuzz.